capstone/src/main/java/com/google/univiz/api/SuggestionData.java

```diff
@@ -0,0 +1,20 @@
+package com.google.univiz.api;
+
+/** SuggestionData represents the college data specific to the Search feature */
+public class SuggestionData {
```

Comment:
Why not use an @AutoValue?

Reply:
+1

Reply:
I decided not to do that in this PR since I just needed a simple prototype. However, I will be adding all the SuggestionData changes I made here: https://github.com/googleinterns/univiz/pull/84 to this PR